### PR TITLE
Setup help tab on settings page

### PR DESCRIPTION
**Pull request by EvanHerman**
This PR adds a new 'Help' tab to the GoDaddy email marketing plugin so users don't have to leave their site to get help to frequently asked questions.

**Example:**
Example

### CODE REVIEW

**Comment by jonathanbardo** [comment]
@fjarrett We got the test passing. Please review.

File: includes/settings.php [hunk 1]
```diff
@@ -488,6 +508,56 @@ public function do_settings_sections( $page, $columns = 2 ) {
...
+			case 'el' :
+
+				$subdomain = 'gr'; // Greek (Ελληνικά)
+
+				break;
+
+		}
+
+		?>
+		<iframe src="<?php echo esc_url( "https://{$subdomain}.godaddy.com/help/managed-wordpress-1000021" ) ?>" frameborder="0" scrolling="no"></iframe>
```
Comment (jonathanbardo) [inline includes/settings.php]: @EvanHerman This is not the right URL I think. We might want to send them to this one: https://www.godaddy.com/help/godaddy-email-marketing-1000013#nav-1. Don't forget to update the unit test 😉

**Comment by EvanHerman** [comment]
@fjarrett This is working well and can be reviewed. I am having some issues resolving the conflict with the dev-lib submodule.

**Comment by fjarrett** [comment]
@EvanHerman Resolved with `git merge develop -X theirs`

**Pull request merged**